src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private static final SimpleDateFormat SIMPLE_DATE_FORMAT = new SimpleDateFormat("dd.MM.yyyy");
+    private static final int DATE_POSITION = 0;
+    private static final int NAME_POSITION = 1;
+    private static final int HOURS_POSITION = 2;
+    private static final int MONEY_POSITION = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        Employee[] employees = new Employee[names.length];
+        Date dateFromData = convertStringToDate(dateFrom);
+        Date dateToData = convertStringToDate(dateTo);
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/229344775-736304c3-8446-46a1-b25b-092bd609b86c.png)

Reply:
Commit is solved